L1Trigger/L1TTrackMatch/python/L1TkMuonProducer_cfi.py

```diff
@@ -49,3 +49,7 @@
 
 
 )
+
+L1TkMuonsDynamicWindows = L1TkMuons.Clone(emtfMatchAlgoVersion='DynamicWindows')
```

Comment:
cmsRun complains at this (Clone vs clone). I used:
L1TkMuonsDynamicWindows = L1TkMuons.clone() 
L1TkMuonsDynamicWindows.emtfMatchAlgoVersion=cms.string('DynamicWindows')